scripts/perf2perfetto/src/lib.rs

```diff
@@ -0,0 +1,734 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// SPDX-FileCopyrightText: Copyright 2024 Michał Chojnowski <michal.chojnowski@scylladb.com>
```

Comment:
> We can only accept code under the company copyright (or from someone with a CLA).

That's not true, we have cassandra-copyrighted code in the repository.

And what does copyright matter, if it's licenced under GPL? 

> Please use AGPL.

I'm not sure I'm allowed to, because that `perf-dlfilter.h` header I'm using is licenced under `GPL-2.0`, not `GPL-2.0+`.

Reply:
And even if we were distributing it, it isn't a part of Scylla, but a separate binary. Why would it infect anything?

Reply:
This is something you have to explain to someone that runs some automated tool later. A human wouldn't mind.